cmd/stencil/update_module.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2022 Outreach Corporation. All Rights Reserved.
+
+// Description: This file contains code for the create templaterepository command
+
+package main
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/getoutreach/stencil/pkg/configuration"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli/v2"
+	"gopkg.in/yaml.v3"
+)
+
+// NewUpdateModule returns a new urfave/cli.Command for the
+// update module command
+func NewUpdateModule() *cli.Command {
+	return &cli.Command{
+		Name:        "module",
+		Description: "updates a module with the provided name in the current directory",
+		ArgsUsage:   "update module",
+		Flags: []cli.Flag{
+			&cli.BoolFlag{
+				Name:  "--remove-native-extension",
+				Usage: "Updates a module to be a native extension",
```

Comment:
Usage string seems to be wrong, should it be "Removes native extension configuration for the provided module"?